bftengine/src/bftengine/InternalBFTClient.cpp

```diff
@@ -15,15 +15,25 @@
 #include "chrono"
 #include "Logger.hpp"
 
+#include <utility>
+
 uint64_t InternalBFTClient::sendRequest(uint64_t flags,
                                         uint32_t requestLength,
                                         const char* request,
                                         const std::string& cid) {
+  return sendRequest(flags, requestLength, request, cid, std::function<void()>{});
```

Comment:
in [https://github.com/vmware/concord-bft/pull/1736](url) you defined ```using Callback = std::function<void()>;``` maybe it's worth using it

Reply:
Sure, I will do that.